packages/calcite-components/src/tests/utils.ts

```diff
@@ -213,6 +235,8 @@ export async function visualizeMouseCursor(page: E2EPage): Promise<void> {
     document.addEventListener(
       "mousemove",
       (event) => {
+        event.preventDefault();
```

Comment:
Why is this canceling the event?

Reply:
Oops, added them as part of debugging, will remove.